recipes/cytoolz/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.7.5" %}
+
+package:
+  name: cytoolz
+  version: {{ version }}
+
+source:
+  fn: cytoolz-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/c/cytoolz/cytoolz-{{ version }}.tar.gz
+  md5: 15fbc1c56fca80cbca0e89f37c93d432
+
+build:
+  number: 0
+  script: python setup.py install --record=record.txt
+
+requirements:
+  build:
+    - setuptools
+    - python
+    - cython
+    - toolz
+    - libpython  # [win and not (win64 and py34)]
```

Comment:
libpython won't work in general.  It's used mostly for people who use GCC as their compiler, to be able to link with things compiled with MSVC.  There is presently no libpython available for Python 3.5 / VS 2015.

Assuming all things here are compiled from source, you should not need libpython - all of our stuff is MSVC-only.